docs/extend/plugins_services.md

```diff
@@ -0,0 +1,183 @@
+---
+description: Using services with plugins
+keywords: "API, Usage, plugins, documentation, developer"
+title: Plugins and Services
+---
+
+<!-- This file is maintained within the docker/docker Github
+     repository at https://github.com/docker/docker/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# Service creation using plugin filters.
```

Comment:
If this is the only existing documentation to introduce the plugin-v2 concept being used along-side services, then the title should not be narrowly focussing on `filters`.
IMO, it should be more generic such as `Volume and Network plugins for Docker services` (also the existing title is a bit misleading ... the plugins themselves doesnt assist the service creation. The plugins are merely drivers that replace the default drivers).